File: 2413/main.go
package main

import "fmt"

func main() {
	fmt.Println(smallestEvenMultiple(5))
}

func smallestEvenMultiple(n int) int {
	if n % 2 == 0 {
		return n
	} else {
		return 2 * n
	}
}

File: 2413/main.rs
struct Solution {}

impl Solution {
    pub fn smallest_even_multiple(n: i32) -> i32 {
        if n % 2 == 0 {
            return n
        } else {
            return 2 * n
        }
    }
}

fn main() {
    println!("{}", Solution::smallest_even_multiple(5))
}